clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -2923,6 +2923,18 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
       SetSqrtFPAccuracy(Call);
       return RValue::get(Call);
     }
+
+    case Builtin::BItan:
```

Comment:
What is the difference between `BItanf` and `BI_builtin_tanf` ?

Reply:
def TanF16F128 : Builtin, F16F128MathTemplate = BI_builtin_tanf
vs 
def Tan : FPMathTemplate, LibBuiltin<"math.h">  = BItanf